custom_components/tuya_local/devices/solaire_vitra_wifi_electric_panel_heater.yaml

```diff
@@ -0,0 +1,72 @@
+name: Sol*Aire Vitra WiFi Electric Panel Heater
+primary_entity:
+  entity: climate
+  dps:
+    - id: 1
+      name: hvac_mode
+      type: boolean
+      mapping:
+        - dps_val: true
+          value: "heat"
+          icon: "mdi:radiator"
+        - dps_val: false
+          value: "off"
+          icon: "mdi:radiator-disabled"
+    - id: 2
+      name: temperature
+      type: integer
+      range:
+        min: 7
+        max: 30
+    - id: 3
+      name: current_temperature
+      type: integer
+    - id: 4
+      name: preset_mode
+      type: string
+      mapping:
+        - dps_val: eco
+          value: eco
+        - dps_val: comfort
+          value: comfort
+        - dps_val: antifreezen
+          value: away
+    - id: 12
+      name: error
+      type: bitfield
+      mapping:
+        - dps_val: 0
+          value: "OK"
+secondary_entities:
+  - entity: number
+    name: Temperature calibration
+    category: config
+    icon: "mdi:thermometer-lines"
+    dps:
+      - id: 103
+        type: integer
+        name: value
+        range:
+          min: -15
+          max: 15
+  - entity: lock
+    translation_key: child_lock
+    category: config
+    dps:
+      - id: 6
+        type: boolean
+        name: lock
+        mapping:
+          - dps_val: true
+            value: true
+            icon: "mdi:account-lock"
```

Comment:
It is not necessary to override the icons here.  You are free to suggest a change in icons.json, but I don't see a strong case for the change unless there are multiple other integrations using these icons.

Reply:
These icons more closely match the equivalent function icons in the Smart Things app, but I can appreciate consistency for functions across devices within Home Assistant so happy to leave those as default.
I noticed that the hvac_mode icons are overridden in the andersson_gsh_heater.yaml that got me started figuring out this heater, should they be taken out and default (climate?) icon used?
Thanks again for any help.

Reply:
I haven't yet defined icon translations for common climate subcategories, so they are still using the legacy method to customise.  I think at least the default hvac_off icon probably doesn't match heaters well, so have not removed those yet.